validator/tests/test_context_manager/tests.py

```diff
@@ -211,6 +211,182 @@ def test_address_enforcement(self):
                 address_value_list=[{invalid_address1: b'1'},
                                     {invalid_address2: b'2'}])
 
+    def test_exception_on_invalid_input(self):
+        """Tests that invalid inputs raise an exception. Tested with invalid
+        characters, odd number of characters, and too long namespace;
+
+        Notes:
+            1) Assert that inputs with a namespace with an odd number of
+               characters raises a CreateContextException.
+            2) Assert that inputs with a 71 character namespace raises a
```

Comment:
Grammar - subj./verb agreement for plural vs. singular. Suggestion:
"Assert that inputs with a 71 character namespace raise a..."